js/parts-more/PackedBubbleSeries.js

```diff
@@ -1123,14 +1172,30 @@ seriesType('packedbubble', 'bubble',
      * @param {global.Event} event Browser event, before normalization.
      * @param {Highcharts.Point} point The point that event occured.
      */
-    onMouseDown: dragNodesMixin.onMouseDown,
+    onMouseDown: function (point, event) {
+        var isParentNode = point.isParentNode, parentNodeLayout = this.parentNodeLayout;
+        if (isParentNode) {
+            parentNodeLayout.setMaxIterations();
+            parentNodeLayout.currentStep = 0;
+            parentNodeLayout.step();
+        }
+        dragNodesMixin.onMouseDown.apply(this, arguments);
+    },
     /**
      * Mouse move action during drag&drop.
      * @private
      * @param {global.Event} event Browser event, before normalization.
      * @param {Highcharts.Point} point The point that event occured.
      */
-    onMouseMove: dragNodesMixin.onMouseMove,
+    onMouseMove: function (point, event) {
+        var isParentNode = point.isParentNode, parentNodeLayout = this.parentNodeLayout;
+        if (isParentNode) {
+            parentNodeLayout.setMaxIterations();
+            parentNodeLayout.currentStep = 0;
+            parentNodeLayout.step();
+        }
+        dragNodesMixin.onMouseMove.apply(this, arguments);
+    },
```

Comment:
As we use the same code in onMouseMove and onMouseDown, maybe we can move them to separate method?

Reply:
You are absolutely right, I will fix it.